cloudhub-2/modules/ROOT/pages/ch2-deploy-api.adoc

```diff
@@ -178,6 +178,6 @@ Updates that include unavailable ports fail. Successful deployments occupy the n
 
 == Access Your TCP App
 
-You can access the app using VPN or a transit gateway and within the private space. Because the apps run on worker nodes, they cannot be accessed directly over the internet. The URL to access a TCP app has this structure: `{application-url}.{environment}.tcp.cloudhub.io:{port}`
+You can access the app using VPN or a transit gateway and within the private space. Because the apps run on worker nodes, they cannot be accessed directly over the internet.  TCP endpoints use wildcard domain. The app-name or prefix is irrelevant. You can access applications on the endpoint: `*.<domain>.<region>.tcp.cloudhub.io:<port>`. 
```

Comment:
```suggestion
You can access the app using VPN or a transit gateway and within the private space. Because the apps run on worker nodes, they cannot be accessed directly over the internet. TCP endpoints use wildcard domains. The app-name or prefix is irrelevant. You can access applications on the endpoint: `*.<domain>.<region>.tcp.cloudhub.io:<port>`. 
```